hw/mcu/dialog/da1469x/src/arch/cortex_m33/da1469x_m33_sleep.S

```diff
@@ -94,8 +94,8 @@ da1469x_m33_sleep:
     add     r1, r0, NVIC_IPR_OFFSET
     ldmia   r0!, {r4-r5}
     stmia   r3!, {r4-r5}
-    ldmia   r1!, {r2, r4-r12}
-    stmia   r3!, {r2, r4-r12}
+    ldmia   r1!, {r2}
```

Comment:
There is something wrong here.
Only **IPR[0..3]** is saved here (and restored later).
If it is intentional `space 44` could be `space 4`.
But most likely those line should be changed to copy one more word, R0 could be used for this as it's current value will not be used later.
```
ldmia   r1, {r0, r2, r4-r12}
stmia   r3!, {r0, r2, r4-r12}
```